lib/util/index.ts

```diff
@@ -2,7 +2,7 @@ import {
   EndpointOption,
   JSONObject,
   SelectOption,
-  SupportedParachain,
+  SupportedParachain
```

Comment:
I think our prettier should include the rule for trailing commas, makes diffing more smooth